buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+repositories {
+    google()
+    maven(url = "https://plugins.gradle.org/m2/")
+    mavenCentral()
+    jcenter()
+}
+
+plugins {
+    java
+    `kotlin-dsl`
+}
+
+dependencies {
+    implementation(gradleApi())
+    // Gradle version can be a lower minior version compared to the main lib/app Gradle project
```

Comment:
```suggestion
    // Gradle version can be a lower minor version compared to the main lib/app Gradle project
```

Reply:
It would be nice to keep the gradle and kotlin plugin versions the same as the rest of the app just for ease of updating

Reply:
I was able to share the values using a properties file https://github.com/airbnb/MvRx/blob/5969f3c8425229e8a607eab28acf0539c6d1626d/versions.properties#L1-L2

The only drawback with this method is a little duplication with loading the file and a few @JvmStatic so Groovy can find the function.